test/core/gc/array_bulk.wast

```diff
@@ -0,0 +1,354 @@
+;; Bulk instructions
```

Comment:
Yeah, it's unfortunate we already violate the structure with the other bulk instructions. We should fix that eventually. It's better to avoid overly monolithic tests that prevent complex instructions from being tested individually.

Similarities aren't really an argument, they exist for other groups of instructions, too.

Reply:
ok, I've split the tests